docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace.md

```diff
@@ -142,13 +143,20 @@ After that a new entry called "jabref \[run\]" will appear in the run configurat
 
 You should use IntelliJ IDEA's internal build system for compiling and running JabRef during development, because it is usually more responsive. Thereby, **it's important** that you understand that JabRef relies on generated sources which are only build through Gradle. Therefore, to build or update these dependencies you need to run the `assemble` Gradle task at least once.
 
+Note that these steps do not work on IntelliJ 2020.2.
+
 To use IntelliJ IDEA's internal build system when you build JabRef through **Build \| Build Project** or use the provided "JabRef Main" run configuration, ensure that
 
 * In **File \| Settings \| Build, Execution, Deployment \| Build Tools \| Gradle** the setting "Build and run using" and "Test using" is set to "IntelliJ IDEA".
-
 * Ignore the Gradle project "buildSrc" by clicking the button **Select Project Data To Import** in the Gradle Tool Window and unchecking the folder "buildSrc".  
-
    ![Ignore the Gradle project &quot;buildSrc&quot;](../.gitbook/assets/intellij-gradle-config-ignore-buildSrc.png)
+* Add the folder `src/main/gen` as generated sources root  
```

Comment:
Shouldn't this be picked-up automatically when you import the gradle project? If so, I would phrase it like "Make sure that the folder ... is indeed set as ...".

Reply:
It **always** happened, that it is not added. Maybe, we will fix that sometime in our build script (or by a gradle update). Will rephrase.